bubble_sort_by.rb

```diff
@@ -0,0 +1,28 @@
+def bubble_sort_by(array)
```

Comment:
You are not returning the array in this method. When you pass a block like { |x, y| x - y} it returns a number instead of the sorted array.